src/styles/components/_code-pattern.scss

```diff
@@ -16,16 +16,27 @@
     display: grid;
     gap: 1rem; // sass-lint:disable-line no-misspelled-properties
     grid-template-columns: 1fr 1fr;
+    // grid-template-columns: minmax(0, 1fr) minmax(0, 1fr)
   }
 
   .code-pattern__assets,
   .code-pattern__demo {
     border: 1px solid $BORDER_COLOR;
   }
 
+  web-tabs:not(:defined) {
+    display: none;
```

Comment:
Does this still cause a flash of unstyled content?

Reply:
No, this solves it (wrapper has a defined height and content is hidden).